frontend/public/global.css

```diff
@@ -65,4 +65,9 @@ button:focus {
 .error {
   background-color: #f44336 !important;
   color: white;
+}
+
+img:not([data-is-complete="true"]) {
```

Comment:
This selector is too general, it was making every image that did not have the data-is-complete="true" gray.